bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/sipclient.js

```diff
@@ -0,0 +1,15 @@
+import { pt, pb } from '../helpers.js'
+
+export default () => [
+  pt('height', 'Component height', 'Height of the whole component in pixels'),
```

Comment:
I am actually not sure if that is required.

What I though when I implemented that, is that I want to control the appearance/height of the component in my custom widgets to be a large as possible in that custom widget. I am honestly not sure how it looks with automatic resizing, but I will try this out this evening.

Reply:
I checked, the `height` option is required, otherwise the icons are really small.

![image](https://user-images.githubusercontent.com/73423173/180578130-cfde0e9a-84c8-4a02-adf9-8ebc2f689087.png)

Reply:
Right, but no other standard widget uses a custom height parameter, so i guess i would ask why would this be any different ? I can't tell from the screen shot above, but is that using the SIP card widget or using the SIP component by itself in something else?   I guess other components do allow you to specify both a custom image and that image size in PX, maybe we should do that? I think setting the component height will go against the rest of the application design.

Reply:
> but is that using the SIP card widget or using the SIP component by itself in something else?

This is using the SIP card widget.

> I guess other components do allow you to specify both a custom image and that image size in PX, maybe we should do that?

I'm not sure what you mean with image, if you mean the phone icons, I wouldn't allow to change them as that does not really make sense to me. If you mean the image for the video call preview, that's something I would do. 

Talking about size, I would rename the `height` parameter to `iconSize` to follow the standard (e.g. see https://www.openhab.org/docs/ui/components/oh-button.html#iconSize). This would make it more consistent.
The `iconSize` does change the height of the component, otherwise the icons would be cut off.
WDYT?

Reply:
> I'm not sure what you mean with image, if you mean the phone icons

yes i am talking about allowing to set the size of the icon in PX, which is a common pattern in many widgets.  I think that would be enough here.

Reply:
Okay, then I will implement this 👍

Reply:
Done in https://github.com/openhab/openhab-webui/pull/1453/commits/7087dd082ecc6e472137e1405db007e890a25d03